### PR TITLE
Create teamcharter

### PR DESCRIPTION
**Pull request by mariahkannenberg**

### CODE REVIEW

File: docs/teamcharter [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Lia McCabe, User Experience Designer and Researcher
+* Mariah Kannenberg, Delivery Manager
+* Mike Marcotte, Senior Software Developer/Court Tech Lead
+* Mrinalini Sinha, Software Engineer
+* Peter Kendall, Software Engineer
+* Rachel Schneiderman, Software Engineer
+* Rosie Highsmith, Software Engineer
+* Sui Wong, User Experience Design and Research
+* Tenille Lenard, Court Product Specialist
+* Timothy Shumway - Software Engineer
```
Comment (rachelschneiderman) [inline docs/teamcharter:25]: Nit: Comma here instead of '-'?

File: docs/teamcharter [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Voting - As a team we will use multiple methods of voting for decision making within the team. The team will use Roman voting for quick feedback from the group on a singular topic. The team will use dot voting for discussions that have a variety of options to choose from in order to narrow down to a smaller list of actionable items.  Planning poker will be the tool we will use for estimation voting on the urgency and importance in regards to the DevEx/OpEx backlog.
+* Verbal Communication - Do your share of listening, and then verbally ask questions when you don’t understand.  Use the hand raising mechanism in Zoom to draw attention to your question.  Verbally checking for understanding is better than assuming.  Verbalize your concerns because others feel the same way.  Sometimes verbal communication doesn’t work, so try other communication mechanisms.
+
+Technical Working Agreement
+* Smoke and integration tests are accurate to real life scenarios
```
Comment (rachelschneiderman) [inline docs/teamcharter:45]: Did we agree to take this out? I can't remember....

**Review by rachelschneiderman — APPROVED** [review]

File: docs/teamcharter (hunk 2 above)
Comment (mariahkannenberg) [reply docs/teamcharter:45]: I remember you saying to make it less specific, but it seems the language wasn't updated. I'll make a note to bring it up at our next retro.

File: docs/teamcharter (hunk 1 above)
Comment (mariahkannenberg) [reply docs/teamcharter:25]: Updated - good catch!

**Review by mariahkannenberg — COMMENTED** [review]
Thanks for the feedback, @rachelschneiderman !

**Pull request merged**